samples/ChatWithTools/Program.cs

```diff
@@ -6,16 +6,13 @@
 // Connect to an MCP server
 Console.WriteLine("Connecting client to MCP 'everything' server");
 var mcpClient = await McpClientFactory.CreateAsync(
-    new()
+    new StdioClientTransport(new()
     {
         Id = "everything",
         Name = "Everything",
```

Comment:
Are Id and Name used for anything other than diagnostics? It'd be nice if they were no longer required. I'm also not sure what the difference is between the two.

Reply:
My thoughts exactly: https://github.com/modelcontextprotocol/csharp-sdk/pull/230#discussion_r2031719601